### PR TITLE
Updated parser comments

### PR DESCRIPTION
**Pull request by MinickLS**

### CODE REVIEW

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderTSL_1128_UHF.js [hunk 1]
```diff
@@ -3,20 +3,18 @@ var bluetooth = require('./CommBluetooth');
...
-
 /**
- * Provides a parser to parse the RC522 LF Bluetooth device. This calls
- * the successful every time a "frame" has been built, the error callback 
- * if it failed to connect to the device, and the reader object sent in from
- * RadLib.js. On each successful callback, an object with the following properties
+ * Provides a parser to parse the TSL 1128 Bluetooth reader. This calls
+ * success every time an "EP:" tag line is found and the error callback 
+ * if it has failed to connect to the device or after two error reads.
+ * On each successful callback, an object with the following properties
```
Comment (jeslin) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderTSL_1128_UHF.js:10]: No

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderArduino_RC522_LF.js [hunk 2]
```diff
@@ -5,16 +5,16 @@ var ReaderArduino_RC522_LF = {};
 
 /**
  * Provides a parser to parse the RC522 LF Bluetooth device. This calls
- * the successful every time a "frame" has been built, the error callback 
+ * the success every time a "frame" has been built, the error callback
```
Comment (kaysabelle) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderArduino_RC522_LF.js:8]: I noticed this too. YAY GLAD YOU FIXED IT

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderArduino_RC522_LF.js [hunk 3]
```diff
@@ -5,16 +5,16 @@ var ReaderArduino_RC522_LF = {};
...
  * if it failed to connect to the device, and the reader object sent in from
  * RadLib.js. On each successful callback, an object with the following properties
  * is returned:
- *       id id of the read tag
- *       reader device name of this reader
- *       friendlyName user specified name of this reader
- *       time time frame was read
- *       date date frame was read
- *       frame the frame which was parsed
+ *       id - id of the read tag
```
Comment (kaysabelle) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderArduino_RC522_LF.js:12]: :+1:  this is so awesome. good job. ql+ (quality of life++)

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderTSL_1128_UHF.js [hunk 4]
```diff
@@ -3,20 +3,18 @@ var bluetooth = require('./CommBluetooth');
 var resources = require('./Resources');
 var ReaderTSL_1128_UHF = {};
 
-//DARREN FIX THIS COMMENT
-
 /**
```
Comment (kaysabelle) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderTSL_1128_UHF.js:6]: //DARREN FIXED THIS COMMENT

:+1:

**Comment by kaysabelle** [comment]
Wow this pull request was amazing

10/10 would review again A++++++++++++++++++

**Pull request merged**